package-lock.json

```diff
@@ -0,0 +1,7938 @@
+{
```

Comment:
Is this file necessary at all? It seems as if you've ran `npm install` command in project's root folder which generated it. All the contents related to frontend app are in menelaos-front directory, so, IMO that's the only place for package-lock.json. Please elaborate on that.

Reply:
Deleted this file.